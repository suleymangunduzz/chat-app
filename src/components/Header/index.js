import React from 'react';

const Header = () => {
    return (
        <div className="header">
            <div className="header__title">Simple Chat App</div>
            <div className="header__settings">Settings</div>
        </div>
    );
}

export default Header;
