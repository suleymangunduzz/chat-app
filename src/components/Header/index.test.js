import React from 'react';
import renderer from 'react-test-renderer';
import Header from './index';

const mockSetShowSettings = jest.fn();

test('Header component render', () => {
    const component = renderer.create(<Header setShowSettings={ mockSetShowSettings } />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});