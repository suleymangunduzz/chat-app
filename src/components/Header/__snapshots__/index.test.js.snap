// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component render 1`] = `
<div
  className="header"
>
  <div
    className="header__title"
  >
    Simple Chat App
  </div>
  <div
    className="header__settings"
    onClick={[Function]}
  >
    Settings
  </div>
</div>
`;
